apps/core/i18n.ts

```diff
@@ -0,0 +1,33 @@
+import { notFound } from 'next/navigation';
+import { getRequestConfig } from 'next-intl/server';
+
+enum LocalePrefixes {
+  ALWAYS = 'always',
+  NEVER = 'never',
+  ASNEEDED = 'as-needed', // removes prefix on default locale
+}
+
+const locales = ['en', 'de'] as const;
+
+type LocalePrefixesType = `${LocalePrefixes}`;
+
+// Temporary we use NEVER prefix to prioritize accept-language header
+// & disable internationalized routes due to multi-language in Catalyst is in progress
+const localePrefix: LocalePrefixesType = LocalePrefixes.NEVER;
```

Comment:
During request `accept-language` header just helps to detect preferred language & locale for a user but in terms how next-intl handles routing it stays in 3rd place in its list of [priorities](https://next-intl-docs.vercel.app/docs/routing/middleware#prefix-locale-detection):
1. A locale prefix is present in the pathname (e.g. /en/page)
2. A cookie is present that contains a previously detected locale
3. A locale can be matched based on the accept-language header

Most libraries work with internationalized routes out of the box and that’s what we want to get at the end but as Nathan mentioned [here](https://github.com/bigcommerce/catalyst/pull/463#issuecomment-1919780600) not everything can be localised at the moment. And while we’re improving things we can temporary switch our strategy and shift priority from internationalized routes to accept-lang header that is kind of what we’re doing in corner as well. 

Since library can handle lang header we’ve just decided to switch off usage of prefixes in routes to raise up header’s priority so now when a User comes to a store at first time, library takes locale/lang preferences from the header & saves its to cookies and re-uses it later on.

And once we want to change the behaviour, we always can turn back to prefixes in routing with one “click”.